release-scripts/do_release.py

```diff
@@ -207,17 +223,20 @@ def cut_release():
     new_version = release_data["versions"]["version"]
     para("The codebase has been updated to reference the GitHub release "
          "artifacts.")
-    actions()
-    bullet("Add, commit and push the updated files to "
+    bullet("Adding, committing and pushing the updated files to "
```

Comment:
This will require a change to lines 173-181 above also, which already existed like this.